packages/http-client-python/emitter/src/lib.ts

```diff
@@ -83,6 +83,30 @@ const libDef = {
         default: "Can't generate Python SDK since no client defined in typespec file.",
       },
     },
+    "invalid-paging-items": {
+      severity: "warning",
+      messages: {
+        default: paramMessage`No valid paging items for operation '${"operationId"}'.`,
+      },
+    },
+    "invalid-nextlink": {
+      severity: "warning",
+      messages: {
+        default: paramMessage`No valid next link for operation '${"operationId"}'.`,
```

Comment:
smallest nit: `invalid-next-link` since `next-link` is shown as two words

Reply:
Updated